schema/policy.go

```diff
@@ -137,36 +137,64 @@ type resourceFilter struct {
 	Path             *regexp.Regexp
 }
 
+//ScopingType describes how the authorization is scoped
+type ScopingType string
+
+const (
+	ScopedToTenant ScopingType = "to_tenant"
+	ScopedToDomain ScopingType = "to_domain"
+)
+
 //Authorization interface
 type Authorization interface {
 	TenantID() string
 	TenantName() string
+	DomainID() string
+	DomainName() string
 	AuthToken() string
```

Comment:
it looks like `AuthToken` and `Catalog` are never actually read (to be confirmed). Could we remove these fields and any associated logic?